spec/services/fee_scheme_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe FeeScheme do
+  subject(:fee_scheme) { FeeScheme.new('AGFS', "2018-01-01") }
```

Comment:
Linting: prefer single-quotes

```suggestion
  subject(:fee_scheme) { FeeScheme.new('AGFS', '2018-01-01') }
```